addon/service.js

```diff
@@ -19,8 +19,20 @@ export default Ember.Object.extend(Ember.Evented, {
   t: function(key, data = {}) {
     const locale = this.get('_locale');
     Ember.assert("I18n: Cannot translate when locale is null", locale);
+    const fallbacks = get(data, 'fallbacks');
+    const count = get(data, 'count');
 
-    const template = locale.getCompiledTemplate(key, get(data, 'count'));
+    let template = locale.getCompiledTemplate(key, count);
+
+    if (template._isMissing && fallbacks) {
+      for (let i = 0; i < fallbacks.length; i++) {
```

Comment:
Oh, except that might end up with `undefined`, so you'd then need to pick the last fallback. That's not really any nicer :/